### ISSUE TITLE
IOU

### ISSUE THREAD

**Issue by Azure333** [hvcl/ScribbleCont#2]
I'm interested in your project , I've run through your code with my dataset, but I have a question, your model solves the overfitting problem, why the accuracy is not good.

**Comment by hjoh01** [comment]
Could you please clarify what you mean? Are you referring to the accuracy of the model on your dataset or something specific about how overfitting is addressed? I'd like to understand better so I can assist you more effectively.

**Comment by Azure333** [comment]
Hello, thank you very much for your reply. The current model has already solved the overfitting problem, but the accuracy of my data on this model has actually decreased.

**Comment by hjoh01** [comment]
Hi, we have found that our approach also get affected by the consistency threshold for pseudo label of S2L. (Currently, we applied it as 0.95 for the MoNuSeg dataset in the paper.)
If you change the consistency threshold (e.g., as 0.5), it may improve the model performance.
To summarize, it requires hyperparameter tuning for the dataset.

**Comment by Azure333** [comment]
Thank you very much. May I ask what exactly hyperparameter optimization is and which threshold it includes?

**Comment by hjoh01** [comment]
I have tried to change the hyperparameter "thr_conf" in the main.py.
Currently, it is samely applied for both L_p (pseudo label loss) and L_c.
But when I changed thr_conf for L_p as 0.5 and froze the UNet model's drop out layers ( by setting unet.to(eval) ) during main stage, I could improve the performance compared to the scores reported in our paper.
If it does not work and have additional questions, please leave a comment again!

**Comment by Azure333** [comment]
Great! Thank you very much for your reply. I'll give it a try.
